From e3d88f59ad497312341559fb308f270e58266609 Mon Sep 17 00:00:00 2001
From: Dale Curtis <dalecurtis@chromium.org>
Date: Mon, 26 Aug 2013 17:47:59 -0700
Subject: [PATCH] Disable unused PIX_FMT_DESC API to avoid deprecation
 warnings.

Chrome doesn't use this API, so disable it until its removed in
a later FFmpeg version.
---
 libavutil/version.h | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/libavutil/version.h b/libavutil/version.h
index e522b79..4d02ea3 100644
--- a/libavutil/version.h
+++ b/libavutil/version.h
@@ -114,9 +114,11 @@
 #ifndef FF_API_CONTEXT_SIZE
 #define FF_API_CONTEXT_SIZE             (LIBAVUTIL_VERSION_MAJOR < 53)
 #endif
-#ifndef FF_API_PIX_FMT_DESC
-#define FF_API_PIX_FMT_DESC             (LIBAVUTIL_VERSION_MAJOR < 53)
-#endif
+// TODO(dalecurtis): Remove when libavutil > 52.  We don't care about this API,
+// but without this it'll generate spammy deprecation warnings.
+// #ifndef FF_API_PIX_FMT_DESC
+// #define FF_API_PIX_FMT_DESC             (LIBAVUTIL_VERSION_MAJOR < 53)
+// #endif
 #ifndef FF_API_AV_REVERSE
 #define FF_API_AV_REVERSE               (LIBAVUTIL_VERSION_MAJOR < 53)
 #endif
--
1.8.3

